jobs/jobs.yml

```diff
@@ -1225,6 +1230,40 @@
 
     <<: *plone-basic
 
+- job:
+    name: exp-parallel-tests
+    display-name: 'Plone 5.2 parallel tests'
```

Comment:
`'Plone 5.2 - Python 3.6 - parallel'`?

Reply:
It is temporary, and as we can easily change the python version, I wouldn't bother to change that.